printer-linter/src/printerlinter/diagnostic.py

```diff
@@ -8,13 +23,12 @@ def __init__(self, file, diagnostic_name, message, level, offset, replacements=N
         self.replacements = replacements
 
     def toDict(self) -> dict:
```

Comment:
incomplete return type
```suggestion
    def toDict(self) -> Dict[str, Any[str, int, List[Replacements]]]:
```

Reply:
Unfortunately Any is not subscriptable like that. I typed it as Dict\[str, Any\]